frontend/views/components/tabs/TabWrapper.vue

```diff
@@ -70,9 +70,25 @@ export default {
       if (this.tabItems.length) {
         this.tabItems[this.activeTab].isActive = true
       }
+    },
+    '$route' (to, from) {
+      console.log('here... just to understand if cypress breaks?')
+      const section = to.query.section
+      if (!section) return
+
+      let foundTheTab = false
+      for (const tabItem of this.tabNav) {
+        for (const link of tabItem.links) {
+          if (this.activeTab !== link.index && link.url === section) {
+            this.changeTab(link.index)
+            foundTheTab = true
+            break
+          }
+        }
+        if (foundTheTab) break
+      }
```

Comment:
@pieer I found out your code to change between tabs (see commit https://github.com/okTurtles/group-income-simple/pull/863/commits/4a5c01f11c1f911eb6abc2add538f23490bf9152) was breaking the tests!

I think the problem was because the code was iterating over `this.tabNav` using `.map()`. I thing in some browsers, if we use `.map()` without returning an array, JS will break. I thing that's what happened on Cypress...

The purpose of `.map()` is creating a new array based on another array and it must return another array. However if we use `.forEach()` we only need to iterate over it (as it is in this case). [That's the core difference between `.map()` and `.forEach()`](https://stackoverflow.com/a/34426481/4737729),

So, I changed the code from `.map()` to `for... of` (faster than `.forEach()`) and everything's fine now.

I also added an early return in case `to.query.section` doesn't exist, to avoid useless iterations.